.github/workflows/backend-deploy.yaml

```
@@ -0,0 +1,80 @@
+---
+name: Build and push backend docker image
+on:
+  push:
+    branches: [build/add-ecr-workflow]
+    paths:
+      - backend/**
+      - terraform/**
+      - .github/workflows/backend-deploy.yaml
```

```suggestion
      - 'backend/**'
      - 'terraform/**'
      - '.github/workflows/backend-deploy.yaml'
```

IIRC single or double shouldn't matter (eg bash), just in case any future additions start with wildcards that the parser thinks are aliases